lib/uid_test.py

```diff
@@ -31,3 +31,69 @@ def test_ensure_uid_unique(self):
         ensure_uid_unique(dfa, ["a"])
         with self.assertRaisesRegex(Exception, r"rows are not unique:"):
             ensure_uid_unique(dfb, ["a"])
+
+class TestCprrPostUidMatching(unittest.TestCase):
+    def test_common_last_name_uid_unchanged(self):
+        natchitoches_df = pd.DataFrame({
```

Comment:
Can you extend this to include examples where we have first names as well for some people? Where the goal is to ensure that we skip entity resolution for persons that A) have no first name and B) a common last name